kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/parquet/ParquetColumnReaders.java

```diff
@@ -77,7 +79,9 @@ public static Converter createConverter(
             return DecimalColumnReader.createDecimalConverter(
                 initialBatchSize, (DecimalType) typeFromClient, typeFromFile);
         } else if (typeFromClient instanceof TimestampType) {
-            return TimestampConverters.createTimestampConverter(initialBatchSize, typeFromFile);
```

Comment:
ditto, but up to you

Reply:
updated the other one. This one has different method calls depending upon the type.